src/GlobalStyles.js

```diff
@@ -0,0 +1,22 @@
+import { createGlobalStyle } from 'styled-components/macro'
+
+export default createGlobalStyle`
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 112.5%;
+  font-family: 'Lato', sans-serif;
+  margin: 50px;
+}
+
+:root {
+    --black-main: #292424;
```

Comment:
Is that really black? The hex code looks grey to me.

Reply:
I updated it to "darkgrey" :-)

Reply:
Done